internal/util/argutil/argutil.go

```diff
@@ -85,3 +88,15 @@ func ParseFieldPath(path string) ([]string, error) {
 	}
 	return newParts, nil
 }
+
+// ResolveSymlink returns the resolved symlink path for the input path
+func ResolveSymlink(ctx context.Context, path string) (string, error) {
+	rp, err := filepath.EvalSymlinks(path)
+	if err != nil {
+		return "", err
+	}
+	if rp != path {
```

Comment:
Just an FYI: One thing I ran into last time when I was using EvalSymlinks was that by default on OSX, `/tmp` is a symlink that points to `/private/tmp`. So if I run EvalSymlinks on any path under `/tmp` for ex. `/tmp/my-dir`, it will resolve to `/private/tmp/dir`.
```
~ $ ls -l /tmp
lrwxr-xr-x 1 root wheel 11 Jan  1  2020 /tmp -> private/tmp
~ $ go run eval-symlink.go /tmp/my-dir
dir:"/tmp/my-dir" and resolvedDir: "/private/tmp/my-dir"
```

Reply:
This check is true for rest of the cases. Worst case is we will print a message which is not big of an issue.